community/fish/post-install

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+if ! grep -q '^/bin/fish$' /etc/shells; then
+    echo /bin/fish >> /etc/shells
+fi
```

Comment:
This isn't something we typically do for shells (We tend to not touch files in this way at all).

Reply:
I'll remove it tonight!